docs/general_overview.md

```diff
@@ -0,0 +1,68 @@
+# General Overview
+
+The different APIs accessible by this gem are:
+
+- [Products](products_api.md)
+- [Stores](stores_api.md)
+- [Categories](categories_api.md)
+
+In spite of having each their own specification, they all share some basics listed here.
+
+## Common methods
+
+These methods can be used with any of the APIs:
```

Comment:
How many methods are described below? Isn't only one currently?

Reply:
Would `This method` suit better?